src/common/tests/btree.c

```diff
@@ -1051,70 +1049,15 @@ main(int argc, char **argv)
 
 	/* start over */
 	optind = 0;
-
-	while ((opt = getopt_long(argc, argv, "tmC:Deocqu:d:r:f:i:b:p:",
-				  btr_ops, NULL)) != -1) {
-		tst_fn_val.optval = optarg;
-		tst_fn_val.input = true;
-		switch (opt) {
-		case 'C':
-			rc = run_btree_open_create_test();
-			break;
-		case 'D':
-			tst_fn_val.input = true;
-			rc = run_btree_close_destroy_test();
-			break;
-		case 'o':
-			tst_fn_val.input = false;
-			tst_fn_val.optval = NULL;
-			rc = run_btree_open_create_test();
-			break;
-		case 'c':
-			tst_fn_val.input = false;
-			rc = run_btree_close_destroy_test();
-			break;
-		case 'e':
-			rc = run_btree_drain_test();
-			break;
-		case 'q':
-			rc = run_btree_query_test();
-			break;
-		case 'u':
-			tst_fn_val.opc = BTR_OPC_UPDATE;
-			rc = run_btree_kv_operate_test();
-			break;
-		case 'f':
-			tst_fn_val.opc = BTR_OPC_LOOKUP;
-			rc = run_btree_kv_operate_test();
-			break;
-		case 'd':
-			tst_fn_val.opc = BTR_OPC_DELETE;
-			rc = run_btree_kv_operate_test();
-			break;
-		case 'r':
-			tst_fn_val.opc = BTR_OPC_DELETE_RETAIN;
-			rc = run_btree_kv_operate_test();
-			break;
-		case 'i':
-			rc = run_btree_iter_test();
-			break;
-		case 'b':
-			rc = run_btree_batch_oper_test();
-			break;
-		case 'p':
-			rc = run_btree_perf_test();
-			break;
-		default:
-			D_PRINT("Unsupported command %c\n", opt);
-		case 'm':
-		case 't':
-			/* handled previously */
-			rc = 0;
-			break;
-		}
-		if (rc != 0)
-			break;
+	stop_idx = argc-1;
+	if(strcmp(argv[1], "--start-test") != 0){
```

Comment:
(style)  space required before the open brace '{'
(style)  space required before the open parenthesis '('